crates/core/src/models/felt.rs

```diff
@@ -1,3 +1,4 @@
+use reth_primitives::rpc::U256 as RpcU256;
```

Comment:
When do we need this type? I think it's a deprecated type (alias for ether-rs, whereas we must only use reth_rpc_types OR reth_primitives

Reply:
My bad, forgot to delete